lib/wallet/error.rs

```diff
@@ -25,6 +27,16 @@ pub struct Electrum {
     message: String,
 }
 
+impl ToStatus for Electrum {
```

Comment:
I think we can actually delete `Electrum` in its entirety. It's a holdover from past usage that no longer exists